sphinx_immaterial/cpp_domain_fixes.py

```diff
@@ -171,13 +177,16 @@ def describe_signature_as_introducer(
 
 
 def _monkey_patch_cpp_ast_wrap_signature_node(
-    ast_type: Any, wrapper_type: Type[docutils.nodes.Element]
+    ast_type: sphinx.domains.cpp.ASTBase, wrapper_type: docutils.nodes.Element
```

Comment:
wrapped_type should have a type of `Type[docutils.nodes.Element]` --- it is passed a node type, not an actual node.